Cargo.toml

```diff
@@ -2,6 +2,7 @@
 resolver = "2"
 
 members = [
+    "c2pa_c",
```

Comment:
I feel a little weird about using the `c2pa_` prefix for this directory since we don't elsewhere in the project. Perhaps rename the directory to `c_api` or something like that?